### ISSUE TITLE
libstdc++.so.6

### ISSUE THREAD

**Issue by dfroger** [dfroger/conda-build-env#1]
A C++ miniconda package is build on `conda-build-env-0.1.0` box.

Trying to use the package on `Ubuntu 12.04` yield to the error:

```
/usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.21' not found (required by /home/vagrant/miniconda/lib/python2.7/site-packages/_dfndarray.so)
```
## Edit

There is actually the same problem when trying to use the package on `conda-build-env-.0.1.0` box (where the package was built):

```
ImportError: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.21' not found (required by /home/vagrant/miniconda/lib/python2.7/site-packages/_dfndarray.so)
```

**Comment by dfroger** [comment]
On Ubuntu 12.04:

```
$ strings /home/vagrant/miniconda/lib/python2.7/site-packages/_dfndarray.so | grep LIBCXX
GLIBCXX_3.4.9
GLIBCXX_3.4.21
GLIBCXX_3.4
```

and:

```
$ strings /usr/lib/x86_64-linux-gnu/libstdc++.so.6 | grep LIBCXX
GLIBCXX_3.4
GLIBCXX_3.4.1
GLIBCXX_3.4.2
GLIBCXX_3.4.3
GLIBCXX_3.4.4
GLIBCXX_3.4.5
GLIBCXX_3.4.6
GLIBCXX_3.4.7
GLIBCXX_3.4.8
GLIBCXX_3.4.9
GLIBCXX_3.4.10
GLIBCXX_3.4.11
GLIBCXX_3.4.12
GLIBCXX_3.4.13
GLIBCXX_3.4.14
GLIBCXX_3.4.15
GLIBCXX_3.4.16
GLIBCXX_DEBUG_MESSAGE_LENGTH
```

On `conda-build-env-0.1.0` box:

```
$ strings /usr/lib64/libstdc++.so.6 | grep LIBCXX
GLIBCXX_3.4
GLIBCXX_3.4.1
GLIBCXX_3.4.2
GLIBCXX_3.4.3
GLIBCXX_3.4.4
GLIBCXX_3.4.5
GLIBCXX_3.4.6
GLIBCXX_3.4.7
GLIBCXX_3.4.8
GLIBCXX_3.4.9
GLIBCXX_3.4.10
GLIBCXX_3.4.11
GLIBCXX_3.4.12
GLIBCXX_3.4.13
GLIBCXX_FORCE_NEW
GLIBCXX_DEBUG_MESSAGE_LENGTH
```

**Comment by dfroger** [comment]
This can be reproduce with the file `main.cxx`:

```
#include <string>
#include <iostream>

using namespace std;

int main()
{
    string s = "foo";
    cout << s << endl;
}
```

compiled and executed like this:

```
$ g++ main.cxx
$ ./a.out
./a.out: /usr/lib64/libstdc++.so.6: version `GLIBCXX_3.4.21' not found (required by ./a.out)
```

**Comment by dfroger** [comment]
Set 10 of this [doc](https://www.vultr.com/docs/how-to-install-gcc-on-centos-6):

```
echo "/usr/local/lib64" > usrLocalLib64.conf
sudo mv usrLocalLib64.conf /etc/ld.so.conf.d/
sudo ldconfig
```

fix the package use on `conda-build-env-0.1.0` box (without re-building the package).

**Comment by dfroger** [comment]
fixed in 0.4.0
